commands/export.go

```diff
@@ -21,6 +21,10 @@ func init() {
 }
 
 func Export(args []string, flags *pflag.FlagSet) error {
+	if !utils.Confirm("Do you really want to dump your whole pick safe?", false) {
+		return errors.New("Aborted as requested")
```

Comment:
Yes it is, take this example:

```sh
set -o pipefail

pick export | jq ..
```

This will happily try to parse `Aborted as requested` as a JSON string if we do not somehow exit with a non-zero code.